config-coral.ini

```diff
@@ -9,6 +9,8 @@ DashboardURL = http://0.0.0.0:8000
 ScreenshotsDirectory = /repo/data/processor/static/screenshots
 EnableSlackNotifications = no
 SlackChannel = lanthorn-notifications
+; OccupancyAlertsTimeout time is measured in seconds (if timeout <= 0 then no occupancy alerts are triggered)
```

Comment:
At first reading this it is not immediately clear to me what this parameter does. I guess that can be solved by documenting the config file parameters in the future. Or maybe we could change the name. What does Timeout refer to? Normally it would be something where you make a request and wait for Timeout seconds before cancelling the request.

If the idea of this is similar to the Logger -> TimeInterval then a similar name would make sense I think. Something like OccupancyAlertsMinInterval ?

Reply:
Yeah interval is also not the word but it does sound better than timeout